toolchain/lex/tokenized_buffer_test.cpp

```diff
@@ -1028,10 +1028,10 @@ TEST_F(LexerTest, TypeLiteralTooManyDigits) {
   code.append(Count, '9');
 
   Testing::MockDiagnosticConsumer consumer;
-  EXPECT_CALL(consumer,
-              HandleDiagnostic(IsSingleDiagnostic(
-                  DiagnosticKind::TooManyDigits, DiagnosticLevel::Error, 1, 2,
```

Comment:
It looks like you're removing the test of `TooManyDigits`, but not the diagnostic itself. Can you please either remove the diagnostic if it's now unreachable, or keep testing it?

Reply:
Yeah, that's fine.